### DISCUSSION TITLE
How could a LineSeries be dotted line and solid line at different part?

### DISCUSSION THREAD

**Discussion by rivenww** [Live-Charts/LiveCharts2#947] [Ideas]
Actually I know how to make a line be dotted or solid in Cartesian Chart. However when I need to make part of one series be dotted and other part be solid, I can't make it.

Is there a way to implement this?

p.s. I draw two series, one solid and one dotted to pretend it to be one mixed series now, but it looks strange xD

For example I wanna something like this

![image](https://user-images.githubusercontent.com/35623575/208394612-bf3f5296-03b6-448c-a1f3-09fb3981e5d1.png)

Thanks for LiveChart2 anyway!

**Comment by beto-rodriguez** [comment]
Hi and thanks for the report.

This is not supported by default by the library, but you should be able to build it with some math.

The idea is to build a custom `DashEffect` where the first dash is long that seems that it is a solid line, then the dashed area has sorter dashes, this will result on what you need.
